D/tests/syntax_test_d.d

```diff
@@ -36,11 +36,15 @@ module foo.a.b1_3;
       +
 //    ^ comment.block.nested.d punctuation.definition.comment.d
  +/
+/++/
+//^^ comment.block.nested.d punctuation.definition.comment.d
```

Comment:
Perhaps add a test right after this with a `- comment` assertion so we are sure it is terminating?

Reply:
Isn't that what the rest of the file is for ;)